internal/v2/README.md

```diff
@@ -1 +1,56 @@
-This directory has implementation for S2Av2 go client.
+**This directory has implementation for S2Av2 go client**
+
+**How example certificates and keys generated in this directory:**
+
+**To create a self signed cert(and private key), Run the following commands using openssl:**
+
+openssl req -x509 -sha256 -days 7305 -newkey rsa:2048 -keyout root_key.pem -out
+root_cert.pem
+
+**To create a chain of certs:**
+
+**leafCert.pem < intermediateCert.pem < rootCert.pem**
+
+**Run the following commands using openssl:**
+
+Create a self signed root:
+
+openssl req -x509 -sha256 -days 7305 -newkey rsa:2048 -keyout root_key.pem -out root_cert.pem
```

Comment:
nit: throughout can we make all of the days parameters 99999, just so we really don't have to worry about expiration in ~10 years

Reply:
Facepalm.

20 years is fine :)